doc/md/admin/upgrading_crossing_0-14-1.md

```diff
@@ -0,0 +1,35 @@
+# Upgrading to 0.14.1 and beyond from any pre-0.14.1 version
```

Comment:
I don't know if it is worth keeping this documentation from such old versions. but you know better than me the history around the component